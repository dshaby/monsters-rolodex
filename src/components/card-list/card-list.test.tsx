import {render} from "@testing-library/react";
import CardList from "./card-list.component";

const mockMonsters = [{
    id: "1",
    name: "Daniel",
    email: "hey@me.com"
}
]

test("Card-List Component", () => {
    const {asFragment} = render(<CardList monsters={mockMonsters}/>);
    expect(asFragment()).toMatchSnapshot();
})