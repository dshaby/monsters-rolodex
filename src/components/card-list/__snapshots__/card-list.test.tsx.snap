// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card-List Component  1`] = `
<DocumentFragment>
  <div
    class="card-list"
  >
    <div
      class="card-container"
    >
      <img
        alt="monster Daniel"
        src="https://robohash.org/123?set=set2&size=180x180"
      />
      <h2>
        Daniel
      </h2>
      <p>
        hey@me.com
      </p>
    </div>
  </div>
</DocumentFragment>
`;

exports[`Card-List Component 1`] = `
<DocumentFragment>
  <div
    class="card-list"
  >
    <div
      class="card-container"
    >
      <img
        alt="monster Daniel"
        src="https://robohash.org/1?set=set2&size=180x180"
      />
      <h2>
        Daniel
      </h2>
      <p>
        hey@me.com
      </p>
    </div>
  </div>
</DocumentFragment>
`;
